### PR TITLE
Clean up. Server starts faster, removed un-used stuff & broken stuff.

### PR DESCRIPTION
**Pull request by xxFlare**

### CODE REVIEW

**Comment by thebigsmileXD** [comment]
Sure, simply remove the TPS stats. great idea.

**Comment by xxFlare** [comment]
@thebigsmileXD What?

SendUsageTask is not needed.

config plugin is broken. whats the point of keeping it there?

And those checkticks only run once on startup, so its useless.

**Comment by thebigsmileXD** [comment]
SendUsageTask should be used, its the I/O of e.g. /status
Config plugin is broken? then fix it. Removing is not a fix, you are not a pokemon go coder.
Autoupdater can be useful though

**Comment by xxFlare** [comment]
@thebigsmileXD Do you understand that this fork is for improved PM performance not features?

**Comment by thebigsmileXD** [comment]
:joy: Not working but performance?

**Comment by daniktheboss** [comment]
😂

**Comment by DelmerHacker** [comment]
PocketMine-Steadfast is Very Easy To Hack With mcpe master You Can Crash the server And Put Lava And Water Mr xFlare!

**Comment by DelmerHacker** [comment]
PocketMine-Steadfast is a AntiHack Plugin eg When you add A Item The server's Crash Go To ItzFreaks You Tube To prove it L b s g You can crash the server Stardfast plugin Has A build of 1.7ZFH It Means AnyCrashHack PocketMine-MP Hack Code On A Server Use it to Hack creative - / Game ode 1 Command IAX go /game mode 1 Cretaive- Hack Code 1.39373973962973 Are Use Ghost hack! just Saying

**Comment by xxFlare** [comment]
@DelmerHacker 
By default, Steadfast protects against many hacks.

Which mods are you using? My team could not reproduce any hacking bugs on our servers.

**Comment by thebigsmileXD** [comment]
@DelmerHacker I can't understand your language, use english please

**Comment by DelmerHacker** [comment]
xFlare.

My Hacks are Client Patches On MCPE They Let Me Use a patch On your Server
Fine My hacks

DelHack! I code Hacks And Use Java Script
Ghost hack You will Get it on PocketMine forms,
Toolbox
SOANHack
DragOP
MCPE, Client Patch!

**Comment by DelmerHacker** [comment]
XFlare


PocketMine-Steadfast Lets Me put lava and Water + I have Client Hacks 

I know All Your Server Hack Codes 

Beacuse I'm a PocketMine HACKER I know all PocketMine Point And Commands To Hack It

Like on Your Server Go / DelHack You Don't have Perm To use This Command All Hack Are made by me

On lifeboat Williamtrd Fogot To Patch In-command Gamemodes Like If You Go To Cretaive Mode It Puts You back To Survival Your Lucky xFlare That Williamtrd [Patched] ITEM Hacks Are Your Server Would Be in danger of A Cyber-Hacker Attack Or a Glicth To Crash Your server 

1. Thing I love about PocketMine steadfast is That If you Add an Item You Can Crash The server 

Eg If you Use MCPE master To add Diamond Sword Enchnated Level 100+ Add It Your Server Crashes Can't take the Commands Just Goes Boom

**Comment by DelmerHacker** [comment]
Nj
image
image
image
image

**Comment by DelmerHacker** [comment]
1. Thing You Guys Don't know is xFlare is VERY Kind And Has a Good Heart I know From FlareHub Hunger XFlare Do you use NOIP Updater!?

File: src/pocketmine/Server.php [hunk 1]
```diff
@@ -92,7 +92,6 @@
 use pocketmine\plugin\PluginManager;
 use pocketmine\scheduler\CallbackTask;
 use pocketmine\scheduler\GarbageCollectionTask;
-use pocketmine\scheduler\SendUsageTask;
 use pocketmine\scheduler\ServerScheduler;
 use pocketmine\tile\Chest;
```
Comment (DelmerHacker) [inline src/pocketmine/Server.php]: Jj

Comment (KairusDarkSeeker) [inline src/pocketmine/Server.php]: what's this for

**Pull request closed without merging**